### PR TITLE
vespa-deploy with required security: athenz and https

### PR DESCRIPTION
**Pull request by bjormel**
This should maybe have been don on a deeper level to obtain the `@configsources` with `$VESPA_HOME/bin/vespa-print-default configservers_http`?

### CODE REVIEW

**Comment by bjormel** [comment]
@bergum, @andreer, @ldalves, @smorgrav, @olaaun FYI

File: container-search/pom.xml [hunk 1]
```diff
@@ -223,7 +223,7 @@
             <arg>-Xlint:-deprecation</arg>
             <arg>-Xlint:-dep-ann</arg>
             <arg>-Xlint:-cast</arg>
-            <arg>-Werror</arg>
+            <!--arg>-Werror</arg-->
```
Comment (hmusum) [inline container-search/pom.xml]: Did you intend to include this change in this PR?

**Review by hmusum — APPROVED** [review]
One comment, LGTM

File: container-search/pom.xml (hunk 1 above)
Comment (bjormel) [reply container-search/pom.xml]: sorry, thanks for catching

**Pull request merged**